WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -137,87 +180,89 @@ final class HubMenuViewModel: ObservableObject {
     }
 }
 
+protocol HubMenuItem {
+    static var id: String { get }
+    var title: String { get }
+    var icon: UIImage { get }
+    var iconColor: UIColor { get }
+    var badge: HubMenuBadgeType { get }
+    var accessibilityIdentifier: String { get }
+    var trackingOption: String { get }
+}
+
+extension HubMenuItem {
+    var id: String {
+        type(of: self).id
+    }
+}
```

Comment:
In this case, since each string has a different purpose, I decided to keep it separate and decoupled. I find that easier to maintain and less error prone in the future; imagine if two implementations have the same tracking id for whatever reason, our functionality here will break.